packages/react/src/components/Authenticator/FederatedIdentity/controls/IdentitiesControl.tsx

```diff
@@ -0,0 +1,17 @@
+import { IdentityControl } from './IdentityControl';
+import { ProviderData } from '../types';
+
+export interface IdentitiesControl<T extends string = string> {
+  (props: ChildrenProps | RenderIdentityProps): JSX.Element;
+  Identity: IdentityControl<T>;
+}
+
+interface ChildrenProps {
+  children?: React.ReactNode;
+  renderIdentity?: never;
```

Comment:
Along the same vein as my other comment but just seeing `renderIdentity` here I feel like it might be hard to understand what you're overriding?

Either way it isnt blocking but something we should think about for API review